scripts/prepdocslib/filestrategy.py

```diff
@@ -61,12 +58,20 @@ async def run(self, search_info: SearchInfo):
             files = self.list_file_strategy.list()
             async for file in files:
                 try:
-                    pages = [page async for page in self.pdf_parser.parse(content=file.content)]
+                    key = file.file_extension()
+                    processor = self.file_processors.get(key)
+                    if not processor:
+                        # skip file if no parser is found
```

Comment:
Given we didnt display skip before, I think we can leave it out now- itd show for every md5 file which would get noisy.